dockerfiles/init/modules/openshift/files/scripts/deploy_che.sh

```diff
@@ -217,6 +229,65 @@ elif [ "${COMMAND}" != "deploy" ]; then
 fi
 
 # -------------------------------------------------------------
+# Starting secondary servers
+# for postgres and optionally Keycloak
+# -------------------------------------------------------------
+
+COMMAND_DIR=$(dirname "$0") 
+
+if [ "$CHE_MULTI_USER" == "true" ]
+then
+    if [ "$CHE_DEDICATED_KEYCLOAK" == "true" ]
+    then
+        "${COMMAND_DIR}"/multi-user/deployPostgresAndKeycloak.sh
+    else
+        "${COMMAND_DIR}"/multi-user/deployPostgresOnly.sh
+    fi
+fi
+
+"${COMMAND_DIR}"/multi-user/wait_until_postgres_is_available.sh
+
+# -------------------------------------------------------------
+# Setting Keycloak-related environment variables
+# Done here since the Openshift project should be available
+# TODO Maybe this should go into a config map, but I don't know
+# How we would manage the retrieval of the Keycloak route
+# external URL.
+# -------------------------------------------------------------
+
+if [ "$CHE_DEDICATED_KEYCLOAK" == "true" ]
+then
+  CHE_KEYCLOAK_SERVER_ROUTE=$(oc get route keycloak -o jsonpath='{.spec.host}' || echo "")
+  if [ "$CHE_KEYCLOAK_SERVER_ROUTE" == "" ]
+  then
+    echo "The dedicated Keycloak server should be started and available through a route before starting the Che server"
```

Comment:
Please prefix this with `[CHE] **ERROR**: ` to be consistent with the rest of the of the script and make it clear that there is an error.

Reply:
The same applies to other `echo` you have added.